### PR TITLE
Add a lambda to mass deploy elastio cfn stack

### PR DESCRIPTION
**Pull request by ivankovnatsky**

### CODE REVIEW

**Review by anelson — COMMENTED** [review]
This seems fine and I assume it's been tested.  But please add a `README.md` with some instructions, background, how to deploy this.  We'll want to share this w/ customers and they'll need to know how to use it.

**Comment by ivankovnatsky** [comment]
> This seems fine and I assume it's been tested. But please add a `README.md` with some instructions, background, how to deploy this. We'll want to share this w/ customers and they'll need to know how to use it.

Yes, I've tested this in our test org. Added a README.

File: elastio-aws-org-elastio-cfn-deployment/README.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
+# elastio-aws-org-elastio-cfn-deployment
+
+With this solution you can in a simple way redistribute elastio cfn deployment
+across your org.
```
Comment (anelson) [inline elastio-aws-org-elastio-cfn-deployment/README.md:4]:
```suggestion
With this solution you can easily bulk-deploy the Elastio CFN across your AWS organization.
```

File: elastio-aws-org-elastio-cfn-deployment/README.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+  - aws cli
+
+## Usage
+
+To deploy the solution run the following commands:
+
+```console
+terraform init
+terraform plan
+terraform apply -auto-approve
```
Comment (anelson) [inline elastio-aws-org-elastio-cfn-deployment/README.md:29]: I don't think we should include `-auto-approve` here.  Let the user review the changes and approve manually.

File: elastio-aws-org-elastio-cfn-deployment/README.md [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+```
+
+## Launching the product
+
+_(Both cli commands are supposed to be run under target account where the
```
Comment (anelson) [inline elastio-aws-org-elastio-cfn-deployment/README.md:34]: Everywhere you have written `cli` replace with `CLI`.  Abbreviations in English are always ALL CAPS.

File: elastio-aws-org-elastio-cfn-deployment/README.md (hunk 1 above)
Comment (oandrukh) [reply elastio-aws-org-elastio-cfn-deployment/README.md]: Done

File: elastio-aws-org-elastio-cfn-deployment/README.md [hunk 4]
```diff
@@ -0,0 +1,94 @@
+# elastio-aws-org-elastio-cfn-deployment
+
+With this solution you can in a simple way redistribute elastio cfn deployment
+across your org.
+
+Terraform code what implements it deploys a service catalog product, which is
+shared across the org and could be used to deploy elastio cfn stack in exiting
```
[the remark this answers is not in the document]
Comment (oandrukh) [reply elastio-aws-org-elastio-cfn-deployment/README.md]: Done

File: elastio-aws-org-elastio-cfn-deployment/README.md [hunk 5]
```diff
@@ -0,0 +1,94 @@
...
+```console
+terraform init
+terraform plan
+terraform apply -auto-approve
+```
+
+## Launching the product
+
+_(Both cli commands are supposed to be run under target account where the
+elastio cfn stack needs to be deployed)_
```
[the remark this answers is not in the document]
Comment (oandrukh) [reply elastio-aws-org-elastio-cfn-deployment/README.md]: Done

File: elastio-aws-org-elastio-cfn-deployment/README.md [hunk 6]
```diff
@@ -0,0 +1,94 @@
...
+## Requirements
+
+- AWS
+  - Control Tower should be configured along with Landing Zone
+  - Make sure to run this code under credentials of the root aws account
+- Tools
+  - Python 3.6 or newer
+  - zip
+  - terraform 0.13 or newer
+  - aws cli
```
[the remark this answers is not in the document]
Comment (oandrukh) [reply elastio-aws-org-elastio-cfn-deployment/README.md]: Done

File: elastio-aws-org-elastio-cfn-deployment/README.md (hunk 3 above)
Comment (oandrukh) [reply elastio-aws-org-elastio-cfn-deployment/README.md]: Replaced

**Pull request merged**